content/en/blog/2023/announcing-notation-rc3.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Notation v1.0.0-RC.3 is available!
+author:  "Notation Release Team"
+date: 2023-03-08
+draft: false
+---
+
+The Notation maintainers are pleased to announce the release of Notation v1.0.0-RC.3, including Notation CLI v1.0.0-rc.3. This blog walks you through the major updates of this release.
+
+## What's new
+
+This release introduces a breaking change to Notation:
+
+- The default type of signature manifest is changed to image manifest. The flag `--signature-manifest` for `notation sign` command is experimental for users to store signatures using artifact manifest.
```

Comment:
If there is only one point per change, maybe it's not necessary to use a single bullet here and combine them into one  paragraph.